test_car.py

```diff
@@ -250,10 +250,10 @@ def test_braking_deceleration_forward(car_object_basic):
     car_object_basic.accelerate_forward(2.5,5)
     car_object_basic.braking_deceleration(1, 3)
     assert car_object_basic.check_velocity() == 9.5
-    assert car_object_basic.check_distance() == 565.75
+    assert car_object_basic.check_distance() == 573.25
```

Comment:
I am getting 564.25 instead of 573.25

Here is my math:
When velocity is 0 and distance is 500, we accelerate forward 2.5 distance for 5 time units.
New Velocity = 2.5 * 5 =   12.5
New Distance = 500 + .5 * 2.5 * 5^2 + 0 * 5 = 531.25

Then I brake 1 distance unit for 3 time units
New velocity = 12.5 - 1 * 3 = 9.5
New distance = 0.5 * (-1) * 3^2 +12.5*3 + 531.25 = 564.25